app/services/TipService.scala

```diff
@@ -0,0 +1,9 @@
+package services
+
+import com.gu.support.workers.model.PaymentMethod
+
+case class PaymentSuccessfulNotification(country: String, contributionType: String, paymentMethod: String)
```

Comment:
3 fields I'd be tempted to split over 3 lines
```suggestion
case class PaymentSuccessfulNotification(
  country: String,
  contributionType: String,
  paymentMethod: String
)
```

Reply:
not to mention that 3 Strings is easy to mix up - consdider using value classes to wrap the strings
e.g. `case class ContributionType(value: String) extends AnyVal`